utils/rest.py

```diff
@@ -46,3 +52,15 @@ def get_vms_in_service(rest_api, service):
     # return entities under /api/vms, not under /api/services/:id/vms subcollection
     # where "actions" are not available
     return [rest_api.get_entity('vms', vm['id']) for vm in service.vms.all]
+
+
+def get_rest_api(appliance, entry_point=None, auth=None, logger=None):
```

Comment:
**REQUIRED** This should be a method of `IPAppliance` that will be used by `IPAppliance.rest_api`.

Reply:
Otherwise it's good to be merged. Feel free to ping me if you want to discuss specifics with regards to that request ^